grpc-server-runtime/src/main/java/io/micronaut/grpc/server/GrpcServerConfiguration.java

```diff
@@ -118,6 +119,22 @@ public int getServerPort() {
         return serverPort;
     }
 
+    /**
+     * The instance id.
+     * @return The instance id
+     */
+    public @Nonnull String getInstanceId() {
+        return instanceId;
+    }
+
+    /**
+     * Sets the instance id.
```

Comment:
Need more clarification in this javadoc what the instance ID is used for (ie service discovery)

Reply:
Done